modules/javafx.graphics/src/main/java/javafx/scene/text/TextFlow.java

```diff
@@ -133,21 +133,21 @@
  * <td>Double.MAX_VALUE</td><td>Double.MAX_VALUE</td></tr>
  * </table>
  * <p>
- * A textflow's unbounded maximum width and height are an indication to the parent that
+ * A {@code TextFlow}'s unbounded maximum width and height are an indication to the parent that
  * it may be resized beyond its preferred size to fill whatever space is assigned to it.
  * <p>
- * TextFlow provides properties for setting the size range directly.  These
- * properties default to the sentinel value Region.USE_COMPUTED_SIZE, however the
+ * {@code TextFlow} provides properties for setting the size range directly. These
+ * properties default to the sentinel value {@code Region.USE_COMPUTED_SIZE}, however the
  * application may set them to other values as needed:
  * <pre><code>
  *     <b>textflow.setMaxWidth(500);</b>
```

Comment:
Maybe remove the `<b>` tag while you are at it?

Reply:
All the other panes use this format, for example, https://openjfx.io/javadoc/19/javafx.graphics/javafx/scene/layout/BorderPane.html.

Maybe in a Panes doc cleanup issue. I always thought more images were needed in these to show the layouts, so I could look into that.